Misc/NEWS.d/next/Tests/2018-09-21-17-33-41.bpo-34537.GImYtZ.rst

```diff
@@ -0,0 +1,2 @@
+Fix test_gdb:test_strings when LC_ALL=C and GDB was compiled with Python 3.6
```

Comment:
I suggest "test_gdb.test_strings()" or "test_strings() of test_gdb". You might use `` around function and file names.